yt_dlp/extractor/common.py

```diff
@@ -1513,6 +1513,24 @@ def _search_nextjs_data(self, webpage, video_id, **kw):
                 webpage, 'next.js data', **kw),
             video_id, **kw)
 
+    def _search_nuxt_data(self, webpage, video_id, context_name='__NUXT__'):
```

Comment:
could u make this a separate PR? (remove redundant code from sovietcloset)

Reply:
okay

Reply:
done #1921